### PR TITLE
docs: oracle upgrade for oracle operators

### PR DESCRIPTION
**Pull request by H4NLee**

### CODE REVIEW

**Review by youngjoon-lee — APPROVED** [review]
lgtm

**Review by audtlr24 — APPROVED** [review]
lgtm

File: .gitbook/5-oracles/1-operate-oracle-nodes/7-oracle-upgrade.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+panacead q oracle oracle-upgrade-info
+```
+
+### Upgrade Oracle Node
+
+{% hint style="info" %}
+You can upgrade oracle any time after an `oracle-upgrade` proposal has passed (even before the upgrade target height is reached).
+However, since the new version of oracle can be active after the target height, the current version of oracle must be run before the target height.
+
+You do not have to stop the old version of oracle when upgrading to a new version of oracle.
```
Comment (inchori) [inline .gitbook/5-oracles/1-operate-oracle-nodes/7-oracle-upgrade.md:45]: Does we need to guide change the `listen-addr`?

**Review by inchori — APPROVED** [review]
LGTM Thanks!

File: .gitbook/5-oracles/1-operate-oracle-nodes/7-oracle-upgrade.md (hunk 1 above)
Comment (H4NLee) [reply .gitbook/5-oracles/1-operate-oracle-nodes/7-oracle-upgrade.md:45]: I think we don't have to. If the same address is used, oracle daemon returns error and exit.
And I expect that they shut down the old version of oracle and start the new one after upgrade.

**Pull request merged**